src/test/java/seedu/address/model/AddressBookTest.java

```diff
@@ -104,49 +103,21 @@ public void getTagList_modifyList_throwsUnsupportedOperationException() {
 
         @Override
         public ObservableList<ReadOnlyPerson> getBlacklistedPersonList() {
-            return getBlacklistedPersons(persons).asObservableList();
+            return persons.stream().filter(person -> person.isBlacklisted())
+                    .collect(Collectors.toCollection(FXCollections::observableArrayList));
         }
 
         @Override
         public ObservableList<ReadOnlyPerson> getWhitelistedPersonList() {
-            return getWhitelistedPersons(persons).asObservableList();
+            return persons.stream().filter(person -> person.isBlacklisted())
```

Comment:
this is wrong.
change to "person.isWhiteListed()"

Reply:
Noted. Changed.